trieste/acquisition/optimizer.py

```diff
@@ -65,16 +75,25 @@ def _discrete_space(space: DiscreteSearchSpace, target_func: AcquisitionFunction
 
 
 @optimize.register
-def _box(space: Box, target_func: AcquisitionFunction) -> TensorType:
-    trial_search_space = space.discretize(tf.minimum(2000, 500 * tf.shape(space.lower)[-1]))
-    initial_point = optimize(trial_search_space, target_func)
+def optimize_continuous(
+    space: Box, target_func: AcquisitionFunction, num_query_points: int = 1
+) -> TensorType:
+    expanded_space = space ** num_query_points
+    trial_search_space = expanded_space.discretize(
+        tf.minimum(2000, 500 * tf.shape(expanded_space.lower)[-1])
+    )
 
-    bijector = tfp.bijectors.Sigmoid(low=space.lower, high=space.upper)
+    def vectorized_target_func(at):
+        return target_func(tf.reshape(at, at.shape[:-1].as_list() + [num_query_points, -1]))
+
+    initial_point = optimize_discrete(trial_search_space, vectorized_target_func)
+
+    bijector = tfp.bijectors.Sigmoid(low=expanded_space.lower, high=expanded_space.upper)
     variable = tf.Variable(bijector.inverse(initial_point))
 
     def _objective() -> TensorType:
-        return -target_func(bijector.forward(variable))
+        return -vectorized_target_func(bijector.forward(variable))
 
     gpflow.optimizers.Scipy().minimize(_objective, (variable,))
-
-    return bijector.forward(variable)
+    points = bijector.forward(variable)
```

Comment:
maybe not part of this PR, but since we're using Scipy it would be better to use L-BFGS-B and avoid using a bijector.

Reply:
yeah defo not part of this PR, but also important